app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -745,7 +745,7 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
             if (presenter.isLockEnabled() && !unlocked)
                 if ((System.currentTimeMillis() > presenter.getSessionExpireMillis())) {
                     blurView.setBlurEnabled(true)
-                    authenticator.authenticate()
+                    authenticator.authenticate(applicationContext.getString(R.string.biometric_title))
```

Comment:
authenticator.authenticate(getString(R.string.biometric_title)) as getString() is available at fragment

Reply:
ah good catch, force of habit here :)